File: src/WithoutStrategyPattern/PaymentApp.java
package WithoutStrategyPattern;

public class PaymentApp {
    // Main Class
    public static void main(String[] args) {
        PaymentSystem system = new PaymentSystem();

        // If user selects CREDIT_CARD
        system.processPayment("CREDIT_CARD", 100);

        // If user selects PAYPAL
        system.processPayment("PAYPAL", 50);
    }
}

File: design-patterns/python/Strategy-DesignPattern/WithoutStrategyPattern/payment_app.py
"""Without strategy pattern demo."""


def main():
    try:
        from .payment_system import PaymentSystem
    except ImportError:
        from payment_system import PaymentSystem

    system = PaymentSystem()
    system.process_payment("CREDIT_CARD", 100)
    system.process_payment("PAYPAL", 50)


if __name__ == "__main__":
    main()
